docs/konflux/release_operator_with_konflux.md

```diff
@@ -39,50 +39,68 @@ Production releases builds on top of the staging releases to do more or less the
 * Release these images to the production repository based on the same snapshot used to release to staging.
 * Release the [FBC](https://docs.openshift.com/container-platform/4.17/extensions/catalogs/fbc.html) (File Based Catalog) fragment to the RH catalog index in production after updating the FBC graph file to include the new fragment like it was done in staging, but using the production image pullspec.
 
+### Preparing the environment
+The release of the orchestrator includes support for multiple versions within the same repository. The git repository for the helm operator defines branches as `release-X.Y` as the location of the semantic X and Y version. In turn, Konflux supports different versions by defining their own uniquely named application, with each containing their own set of compoments, also unique in their name, such as `controller-rhel9-operator-1-2` or `controller-rhel9-operator-1-3`.
+
+Thus, to make it easy and reusable, the release process defined in this page needs to parametrize the names of the components so that the process can be reused as much as possible. We will start the process by defining an environment variable that contains the name of the application that holds the controller and bundle. The next command lists all the applications in the workspace:
+```
+oc get application -ojsonpath='{range .items[*]}{.metadata.name}{"\n"}{end}'
+```
+Example:
+```console
+fbc-v4-13
+fbc-v4-14
+fbc-v4-15
+helm-operator
+helm-operator-1-2
+serverless-workflows
+```
+For our case, we'll use the `helm-operator-1-2` application:
+```console
+applicationName=helm-operator-1-2
```

Comment:
Do we want to update it to -1-3?

Reply:
No need, this is just a reference and it should not match the branch/release where it is hosted. Otherwise we'd have to update it as part of branching for each new release and that's too much work.